test/updateSlice.js

```diff
@@ -0,0 +1,16 @@
+import * as RA from '../src';
+import eq from './shared/eq';
+
+describe('updateSlice', function() {
+  it('should work on strings', function() {
+    eq(RA.updateSlice(1, 3, 'DDDD', 'ABBCCC'), 'ADDDDCCC');
+  });
+
```

Comment:
Missing test for cases like this 
```js
RA.updateSlice(3, 1, 'DDDD', 'ABBCCC')
```
when fromIndex is higher number then to index

Reply:
I'm not sure what's the correct behavior in this case.

Reply:
Let's say that the behavior is unspecified in this usecase. Document it here https://github.com/char0n/ramda-adjunct/pull/410/files#diff-3d5f0b697b4314c9dbc340c8783967b5R8.

We need to add this sentence.

`If toIndex is lower then fromIndex  then the behavior of the updateSlice is unspecified`

What  about the following usecase ?
```js
RA.updateSlice(3, 3, 'DDDD', 'ABBCCC');
```

Reply:
It clearly means insertion, result should be `'ABBDDDDCCC'`.